tutorials/animation/index.md

```diff
@@ -0,0 +1,874 @@
+  object being animated. For example, a Tween might define an interpolation
+  from red to blue, or from 0 to 255.
+* Use Listeners and StatusListeners to monitor animation state changes.
+
+</div>
+
+The animation system in Flutter is based on typed
+[`Animation`](https://docs.flutter.io/flutter/animation/Animation-class.html)
+objects. Widgets can either incorporate these animations in their build
+functions directly by reading their current value and listening to their
+state changes or they can use the animations as the basis of more elaborate
+animations that they pass along to other widgets.
+
+<a name="animation-class"></a>
+### Animation&lt;double&gt;
+
+In Flutter, an Animation object knows nothing about what is onscreen.
+An Animation is an abstract class that understands its current value
+and its state (completed or dismissed). One of the more commonly used
+animation types is Animation&lt;double&gt;.
+
+An Animation object in Flutter is a class that sequentially generates
+interpolated numbers between two values over a certain duration.
+The output of an Animation object may be linear, a curve, a step function,
+or any other mapping you can devise. Depending on how the Animation object
+is controlled, it could run in reverse, or even switch directions in the
+middle.
+
+Animations can also interpolate types other than double, such as
+Animation&lt;Color&gt; or Animation&lt;Size&gt;.
+
+`Animation` object has state. Its current value is always available
+in the `.value` member.
+
+An Animation object knows nothing about rendering or `build()` functions.
+
+### AnimationController
+
+AnimationController is a special Animation object that generates a new
+value whenever the hardware is ready for a new frame. By default,
+an AnimationController linearly produces the numbers from 0.0 to 1.0
+during a given duration. For example, this code creates an Animation object,
+but does not start it running:
+
+<!-- skip -->
+{% prettify dart %}
+final AnimationController controller = new AnimationController(
+    duration: const Duration(milliseconds: 2000), vsync: this);
+{% endprettify %}
+
+AnimationController derives from Animation&lt;double&gt;, so it can be used
+wherever an Animation object is needed. However, the AnimationController
+has additional methods to control the animation. For example, you start
+an animation with the `.forward()` method. The generation of numbers is
+tied to the screen refresh, so typically 60 numbers are generated per
+second. After each number is generated, each Animation object calls the
+attached Listener objects. To create a custom display list for each
+child, see
+[RepaintBoundary](https://docs.flutter.io/flutter/widgets/RepaintBoundary-class.html).
+
+When creating an AnimationController, you pass it a `vsync` argument.
+This option keeps the current widget tree in display memory until
+Flutter's rendering engine completes its refresh cycle. To use your
```

Comment:
The sentence "This option keeps the current widget tree in display memory until Flutter's rendering engine completes its refresh cycle." is wrong.

What 'vsync' does is tie the ticking of the animation controller to the visibility of the widget, so that when the animating widget goes off-screen, the ticking stops, and when the widget is restored, it starts again (without stopping the clock, so it's as if it had been ticking the whole time, but without using CPU).